README.md

```diff
@@ -0,0 +1,88 @@
+# iOS Enhanced media player
+
+iOS media player package. Implemented in `SwiftUI` using `AVFoundation`.
+
+## Table of Contents
+
+- [Motivation](#motivation)
+- [Minimum deployment target](#minimum-deployment-target)
+- [Integration](#integration)
+- [Features](#features)
+- [Visual references](#visual-references)
+
+## Motivation
+
+Internal project focused on learning, training, and practicing media player
+development. Our goal is to build a feature-rich media player, akin to those
+found in commercial platforms like YouTube, using SwiftUI and other contemporary
+development practices. This project serves as a valuable resource for future
+reference, providing hands-on experience in media player creation for iOS platforms.
+
+## Minimum deployment target
+
+iOS 15.0
+
+## Integration
+
+How to integrate with an existing project (SPM or CocoaPod)
+
+Swift Package Manage(SPM):
+
+- Add a new package via github, using the branch `main`
+
+CocoaPods:
+
+- On your project create the `Podfile` bellow
+- Run `pod install`
+
+```ruby
+platform :ios, '15.0'
+
+target 'SampleApp' do
+    pod 'EnhancedMediaPlayer', :git => 'https://github.com/profusion/ios-enhanced-media-player', :
+branch => 'main'
+end
+```
+
+### How to use
+
+The package supports remote and stream video and remote music. It's only
+necessary to provide the URL and the seekFactor. Here is an example of implementation:
```

Comment:
I think it'd be interesting to explain what this `seekFactor` is (I personally don't know)

Reply:
Just added an explanation, thanks